### ISSUE TITLE
Add an option to remove internal solid infill.

### ISSUE THREAD

**Issue by Babsobar** [bambulab/BambuStudio#1229]
**Is your feature request related to a problem? Please describe.**
I print things that have very small and slim walls, with Bambustudio, it will often add solid internal infill in parts that are unneeded.

**Describe the solution you'd like**
A checkbox to disable solid internal infill before slicing so that it is not integrated in the model

**Describe alternatives you've considered**
Using 0% infill and using modeled infill inside the model itself. Far from practical.

I have tried to make myself as clear as possible but english is not my mother tongue so if you have any questions or remarks about this issue don't hesitate

**Comment by SaltWei** [comment]
![image](https://user-images.githubusercontent.com/110660407/217410298-b860791e-84d0-4088-a37b-6cc4db8812b5.png)
Have you tried these two options？

**Comment by Babsobar** [comment]
Yes, here is a screenshot presenting unwanted solid infill with two different setups:
![Solid infill problem2](https://user-images.githubusercontent.com/17145038/217598103-09451ce0-1eed-4363-b2ca-579c2600c8e2.PNG)
![Solid infill problem](https://user-images.githubusercontent.com/17145038/217598138-772840b3-b6a8-4f31-bc71-321170c633db.PNG)

**Comment by SaltWei** [comment]
Could you share this 3MF to us for checking and debugging?

**Comment by Babsobar** [comment]
Github does not allow the share of .3MF files, so here is a download link

[https://we.tl/t-10mzNhSDSY](https://we.tl/t-10mzNhSDSY?utm_campaign=TRN_TDL_05&utm_source=sendgrid&utm_medium=email&trk=TRN_TDL_05)

**Comment by SaltWei** [comment]
![image](https://user-images.githubusercontent.com/110660407/222345964-6d8cf14e-b9d5-4b7b-9112-d36f3763633b.png)
Have you tried set this to be 0?

**Comment by Babsobar** [comment]
Yes, that removes the top of the print, unwanted behavior.

**Comment by DanielSchenk1337** [comment]
same here, there should be an option to deactivate the generation of inill in these areas, e.g. "disable solid internal infill".

How it looks in Bambu Studio:
![Screenshot 2023-07-07 221219](https://github.com/bambulab/BambuStudio/assets/120316900/58c41c87-54a8-4988-8c57-4b6c0d91f88e)


How it should look like: (sliced in Cura)
![Screenshot 2023-07-07 221717](https://github.com/bambulab/BambuStudio/assets/120316900/e67e4d75-d3ff-4cd2-b74f-6ff89cec5954)

Model: https://www.thingiverse.com/thing:1743145  "HIVE_Module_V5.stl"

**Comment by aviatorstengel** [comment]
I am having the same issue when trying to print a building with small outer features. It is adding around an hour and a half to what should be a 4 hour print via Cura. See below.

![image](https://github.com/bambulab/BambuStudio/assets/152718253/4dc5b1fa-e4e9-4a3c-918e-b93af949e6b8)

![image](https://github.com/bambulab/BambuStudio/assets/152718253/992af45b-81ac-407b-9849-123505c4249b)

**Comment by Millagax** [comment]
> EDIT,SOLVED
> 
> **if you have the same problem as in the picture below, set _Minimum sparse infill threshold_ to 0 as SaltWei pointed out**
> 
> How it looks in Bambu Studio(solid infill not wanted):

I'm having the same issue, I want to get rid of solid infill but it's impossible. Someone marked it as solved, but that solution does not work. 

Can someone find a way to deactivate that annoying solid infill?

**Comment by SaltWei** [comment]
> Can someone find a way to deactivate that annoying solid infill?

Could you share a 3mf so we can have a check? Maybe you met different issue with others

**Comment by Millagax** [comment]
3MF file has nothing to do with this issue, besides, another user uploaded it and no aswer was given here. Is is a way to deactivate it or not?
